packages/transformer/src/test/IModelTransformerUtils.ts

```diff
@@ -982,32 +984,38 @@ export class TransformerExtensiveTestScenario extends TestUtils.ExtensiveTestSce
     TransformerExtensiveTestScenario.assertTargetElement(
       sourceDb,
       targetDb,
-      definitionModelId
+      definitionModelId,
+      expectEsas
```

Comment:
`TransformerExtensiveTestScenario.assertTargetElement` is only used within this function. I would recommend just defining it as a local function so it can close over expectEsas and you won't need to add all these new arguments. In fact, you can probably get rid of all but the id parameter if you do that.

Reply:
changed